src/Sylius/Behat/Context/Ui/Shop/AccountContext.php

```diff
@@ -0,0 +1,165 @@
+    private $dashboardPage;
+    
+    /**
+     * @var ProfileUpdatePageInterface
+     */
+    private $profileUpdatePage;
+
+    /**
+     * @var NotificationCheckerInterface
+     */
+    private $notificationChecker;
+
+    /**
+     * @param DashboardPageInterface $dashboardPage
+     * @param ProfileUpdatePageInterface $profileUpdatePage
+     * @param NotificationCheckerInterface $notificationChecker
+     */
+    public function __construct(
+        DashboardPageInterface $dashboardPage,
+        ProfileUpdatePageInterface $profileUpdatePage,
+        NotificationCheckerInterface $notificationChecker
+    ) {
+        $this->dashboardPage = $dashboardPage;
+        $this->profileUpdatePage = $profileUpdatePage;
+        $this->notificationChecker = $notificationChecker;
+    }
+
+    /**
+     * @Given I want to modify my customer profile
+     */
+    public function iWantToModifyMyCustomerProfile()
+    {
+        $this->profileUpdatePage->open();
+    }
+
+    /**
+     * @When I specify the first name as :firstName
+     * @When I remove the first name
+     */
+    public function iSpecifyTheFirstName($firstName = null)
+    {
+        $this->profileUpdatePage->specifyFirstName($firstName);
+    }
+
+    /**
+     * @When I specify the last name as :lastName
+     * @When I remove the last name
+     */
+    public function iSpecifyTheLastName($lastName = null)
+    {
+        $this->profileUpdatePage->specifyLastName($lastName);
+    }
+
+    /**
+     * @When I specify the email as :email
+     * @When I remove the email
+     */
+    public function iSpecifyTheEmail($email = null)
+    {
+        $this->profileUpdatePage->specifyEmail($email);
+    }
+
+    /**
+     * @When I save my changes
+     * @When I try to save my changes
+     */
+    public function iSaveMyChanges()
+    {
+        $this->profileUpdatePage->saveChanges();
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully edited
+     */
+    public function iShouldBeNotifiedThatItHasBeenSuccessfullyEdited()
+    {
+        $this->notificationChecker->checkNotification('has been successfully updated.', NotificationType::success());
+    }
+
+    /**
+     * @Given my name should be :name
+     * @Given my name should still be :name
+     */
+    public function myNameShouldBe($name)
+    {
+        $this->dashboardPage->open();
+
+        Assert::true(
+            $this->dashboardPage->hasCustomerName($name),
+            sprintf('Cannot find customer name "%s".', $name)
+        );
+    }
+
+    /**
+     * @Given my email should be :email
+     * @Given my email should still be :email
+     */
+    public function myEmailShouldBe($email)
+    {
+        $this->dashboardPage->open();
+
+        Assert::true(
+            $this->dashboardPage->hasCustomerEmail($email),
+            sprintf('Cannot find customer email "%s".', $email)
+        );
+    }
+
+    /**
+     * @Then /^I should be notified that the ([^"]+) is required$/
```

Comment:
Not in this case, because I could have `last name` and it wouldn't be match to your sentence